src/filters/reframer.c

```diff
@@ -402,7 +403,22 @@ static Bool reframer_parse_date(char *date, GF_Fraction64 *value, u64 *frame_idx
 		*extract_mode = EXTRACT_RANGE;
 	if (is_dur)
 		*is_dur = GF_FALSE;
+	if (is_timecode)
```

Comment:
You check here for non-NULL but not below. Since the function is not public we don't need to check.

Reply:
I just did the same thing for `is_dur`. I'll remove it.